python/flexflow/torch/model.py

```diff
@@ -1703,7 +1703,7 @@ def __init__(self, node):
     def parse(self):
         s = [self.name]
         scalar = self.innodes[1]
-        if type(scalar) is not int or type(scalar) is not float:
+        if type(scalar) not in [int, float]:
```

Comment:
```suggestion
        if not isinstance(scalar, [int, float]):
```

See discussion in linked issue. I think this would be better but I'm certainly not an expert so if you want to look into it more and convince me the other direction I'm happy to have my mind changed :slightly_smiling_face: